packages/docs/src/page-configs/ui-elements/scroll-container/examples/Horizontal.vue

```diff
@@ -1,5 +1,11 @@
 <template>
-  <va-scroll-container vertical horizontal color="warning" style="max-height: 300px;">
-    <va-image :ratio="4/3" style="width: 2000px; height: 2000px;" src="https://picsum.photos/1500" />
+  <va-scroll-container
+    color="warning"
+    style="max-height: 400px;"
+    horizontal
+  >
+    <div style="min-width: 700px; max-height: 400px;">
+      <img :src="require('@/assets/docs/images/sky.jpeg')" alt="" />
```

Comment:
Doesn't work in the Codesandbox.

Reply:
I think it's not that kind of the example we should provide codesandbox support for :) Images api are poorly suitable here that's why we decided to use custom one.

Reply:
Will try, but there were quality problems - that's why we use custom one.